tests/functional/test_local.py

```diff
@@ -107,7 +107,7 @@ def http_session():
         connect=10,
         # A backoff factor to apply between attempts after the second try.
         backoff_factor=2,
-        method_whitelist=['GET', 'POST', 'PUT'],
+        allowed_methods=['GET', 'POST', 'PUT'],
```

Comment:
Fixes method_whitelist deprecation warning in the next version from urllib3 library

Reply:
Would you mind sending a separate PR for this?  I'd like to keep this one just focused on removing the app.pyi file.